src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -517,6 +517,21 @@ function updateConstSumMessageQn(qnNum) {
                 messageElement.addClass('text-color-green');
                 messageElement.removeClass('text-color-red');
                 messageElement.removeClass('text-color-blue');
+
+                if (distributeToRecipients) {
+                    for (let i = 0; i < numRecipients; i += 1) {
+                        if (isNaN(parseInt($(`#${FEEDBACK_RESPONSE_TEXT}-${qnNum}-${i}-0`).val(), 10))) {
+                            $(`#${FEEDBACK_RESPONSE_TEXT}-${qnNum}-${i}-0`).val(0);
+                        }
+                    }
+                } else {
+                    const recipientIndex = parseInt(messageElement.selector[messageElement.selector.length - 1], 10);
+                    for (let k = 0; k < numOptions; k += 1) {
+                        if (isNaN(parseInt($(`#${FEEDBACK_RESPONSE_TEXT}-${qnNum}-${recipientIndex}-${k}`).val(), 10))) {
+                            $(`#${FEEDBACK_RESPONSE_TEXT}-${qnNum}-${recipientIndex}-${k}`).val(0);
```

Comment:
Would be good to extract the jQuery selectors in both conditionals as a separate variable first. This would help enhance readability, since it's quite difficult to figure out what the selectors mean without having to look at the webpage itself.

Also, perhaps add a comment to explain what this block of code does.